docs/knowledgebase/integrate/encode-txs.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Encoding Transactions
+---
+
+The purpose of this document is to describe the specification for encoding
+[transactions](../../knowledgebase/learn-substrate/extrinsics) to submit to Substrate-based chains.
+
+## Dispatchable Calls
+
+Transactions on Substrate-based chains are used to invoke
+"[dispatchable calls](../../knowledgebase/getting-started/glossary#dispatch)", which are encoded as
+follows:
+
+- 1 byte: `04`, magic number that identifies a dispatchable call
```

Comment:
Not sure where you have that from?

Reply:
It's what happens when you try to reverse engineer an opaque encoding :wink:

Reply:
Why opaque? The encoding is written in our Rust code?

Reply:
I am not sure what the purpose of this discussion is.